explorer/explorerroutes.go

```diff
@@ -1000,20 +1000,28 @@ func (exp *explorerUI) StatsPage(w http.ResponseWriter, r *http.Request) {
 	var stats StatsInfo
 	exp.MempoolData.RLock()
 	stats.TotalSupply = exp.ExtraInfo.CoinSupply
+	if exp.NetName == "Mainnet" {
+		stats.UltimateSupply = 15000000 * 100000000
```

Comment:
What's this from?  There is ultimate supply calculation else where in the dcrdata code you can reference?

Reply:
I couldn't find a reference to the ultimate supply in the code so zaephy gave this option, or could you help me with finding it?

Reply:
Perhaps the naming is not the best: https://github.com/decred/dcrdata/blob/master/txhelpers/subsidy.go#L15

Reply:
Alright, i'll get on that.